### PR TITLE
detailed READMEs for example apps

### PR DESCRIPTION
**Pull request by tracyhenry**

### CODE REVIEW

**Comment by tracyhenry** [comment]
@asah

File: compiler/examples/USMap/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# Installation Instructions for the Impatient
+After running `sudo ./run-kyrix.sh --nba`, open another terminal tab and run the following commands:
+```
+wget https://www.dropbox.com/s/youvfap909mk1m3/usmap_db_psql.sql     # download data
+sudo ./docker-scripts/load-sql.sh usmap_db_psql.sql --dbname usmap   # load data into the database, must be run in root folder
+cp docker-scripts/compile.sh compiler/examples/USMap/compile.sh      # copy the compile script
+cd compiler/examples/USMap                                           # go to the app directory
+chmod +x compile.sh                                                  # make the compile script excutable
+sudo ./compile.sh USMap.js                                           # compile the application
```
Comment (asah) [inline compiler/examples/USMap/README.md:18]: sudo calls in the middle of cut/paste break if user is prompted for root password... another reason to wrap this in a single script...

**Comment by asah** [comment]
tested with USMap - works locally for me, modulo comments added to reviews.

**Comment by tracyhenry** [comment]
> Why copy compile.is instead of running it in place?
> 
> How about a shell script in compiler/examples, which handles all this?
> 
> E.g. compiler/examples/run_example.sh
> 
> `sudo run_example nba_cmv.js`

great point. Let me do this in another PR - will involve some rewriting of docs.

**Pull request merged**